### ISSUE TITLE
New user - compatibility question

### ISSUE THREAD

**Issue by LoZeno** [moo-man/WFRP4e-FoundryVTT#629]
I've installed Foundry for the first time recently (yesterday), specifically to use it for WFRP4e; the version I have installed is 0.8.6, because that's what I can install from the stable channel - I don't see a way to rollback to older versions from within Foundry's UI.
I can't create any new actor - every time I try, I get the following error:

`Cannot assign to  #readonly property 'data' of '#'`

Basically the same issue as https://github.com/moo-man/WFRP4e-FoundryVTT/issues/626

How can I rollback to a compatible version? Or is there a timeline for when version 0.8 will be supported?

**Comment by moo-man** [comment]
You can download older versions by navigating to your license page on foundryvtt.com.

There is no hard timeline, but it may be a while due to the extensive changes in mind.
